### PR TITLE
update + move container + helm release docs

### PR DESCRIPTION
**Pull request by digitaltom**
## Description

* Related Issue / Ticket / Trello card: <link reference>

## Change Type

*Please select the correct option.*

- [ ] **Bug Fix** (a non-breaking change which fixes an issue)
- [ ] **New Feature** (a non-breaking change which adds new functionality)
- [ ] **Documentation Update** (a change which only updates documentation)

## Checklist

*Please check off each item if the requirement is met.*

- [ ] I have reviewed my own code and believe that it's ready for an external review.
- [ ] I have provided comments for any hard-to-understand code.
- [ ] I have documented the `MANUAL.md` file with any changes to the user experience.
- [ ] If my changes are non-trivial, I have added a changelog entry to notify users at `package/obs/rmt-server.changes`.

## Review

Please check out our [review guidelines](https://github.com/SUSE/scc-docs/blob/master/team/workflow/code_review.md) 
and get in touch with the author to get a shared understanding of the change.

### CODE REVIEW

File: PACKAGE.md [hunk 1]
```diff
@@ -106,8 +98,15 @@ You can check the status of your requests [here](https://build.opensuse.org/pack
...
 
-At the moment of writing, the publishing process has to be done manually. This can be achieved by reaching out to the Auto-Build team (only available internally).
+
+#### Helm chart update process
+
+RMT helm chart is defined [here](https://github.com/SUSE/helm-charts.git) and published at `registry.suse.com/suse/rmt-helm`.
+
+Edit `rmt-helm/Chart.yaml` to update the chart version (`version`) and rmt-version (`appVersion`). The `BuildTag` version needs to be updated. Look at this example [pull-request](https://github.com/SUSE/helm-charts/pull/5) bumping the version.
+
+[Bruno Leon](mailto:bruno.leon@suse.com) is the maintainer and point of contact for rmt-helm.
```
Comment (ngetahun) [inline PACKAGE.md:112]: Bruno has handed over the maintenance, but anyone from BCI can help.

**Review by ngetahun — CHANGES_REQUESTED** [review]

File: PACKAGE.md (hunk 1 above)
Comment (digitaltom) [reply PACKAGE.md:112]: What to write here how to contact the BCI team?

Comment (ngetahun) [reply PACKAGE.md:112]: Yes, something like "Please reachout to the BCI team if you have build related questions"

Comment (digitaltom) [inline PACKAGE.md:112]:
```suggestion
Please reach out to the BCI team if you have build related questions.
```

**Review by ngetahun — APPROVED** [review]

**Pull request merged**